deploy/alerts/README.md

```diff
@@ -0,0 +1,23 @@
+# Predefined alerts
+
+These alarming rules come in two files described in the following
+
+## alerts.yaml
+These alarms are triggered by a substantial change (over a certain
+amount of standard deviation). This requires a bit of preparation
+and also some recording rules to be generated.
+
+## simple-comparison.yaml
+
+These alerts are much simpler, as there are basically only comparisons
+of two numbers or percentages.
+
+
+
+## Installation
+
+Log into OpenShift and
```

Comment:
would it make sense to call out the login command?

Reply:
Logging in can vary depending on the environment. If they have made it this far they should already know how to login.